modules/configuration/test/ConfigurationTest.php

```diff
@@ -65,10 +65,14 @@ public function tearDown()
     public function testConfigurationPageLoads()
     {
         $this->safeGet($this->url . "/configuration/");
-
-        $bodyText = $this->webDriver
-            ->findElement(WebDriverBy::cssSelector("body"))->getText();
-        $this->assertContains("Configuration", $bodyText);
+        $bodyText = $this->webDriver->findElement(
+            WebDriverBy::cssSelector("body")
+        )->getText();
+        $bodyText = $this->webDriver->executescript(
+            "return document.querySelector".
+            "('#bc2 > a:nth-child(2) > div').textContent"
+        );
+        $this->assertRegexp("/Configuration/", $bodyText);
```

Comment:
using "Please enter the various configuration variables" instead of "Configuration"